<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1280</width>
                <height>779</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="styleSheet">
            <string notr="true">
                /*去掉item虚线边框*/
                QListWidget, QListView, QTreeWidget, QTreeView {
                outline: 0px;
                }
                /*设置左侧选项的最小最大宽度,文字颜色和背景颜色*/
                QListWidget {
                min-width: 120px;
                max-width: 120px;
                color: black;
                background: white;
                border:none;
                }
                QListWidget::item{
                height:80;
                }
                /*被选中时的背景颜色和左边框颜色*/
                QListWidget::item:selected {
                background: rgb(204, 204, 204);
                border-left: 4px solid rgb(9, 187, 7);
                }
                /*鼠标悬停颜色*/
                HistoryPanel::item:hover {
                background: rgb(52, 52, 52);
                }
            </string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                    <number>0</number>
                </property>
                <property name="leftMargin">
                    <number>0</number>
                </property>
                <property name="topMargin">
                    <number>0</number>
                </property>
                <property name="rightMargin">
                    <number>0</number>
                </property>
                <property name="bottomMargin">
                    <number>0</number>
                </property>
                <item>
                    <widget class="QFrame" name="frame_info">
                        <property name="minimumSize">
                            <size>
                                <width>80</width>
                                <height>500</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>80</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="styleSheet">
                            <string notr="true">background-color:rgb(240,240,240)</string>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::NoFrame</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Plain</enum>
                        </property>
                        <widget class="QLabel" name="myavatar">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>40</y>
                                    <width>60</width>
                                    <height>60</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>avatar</string>
                            </property>
                        </widget>
                        <widget class="QListWidget" name="listWidget">
                            <property name="geometry">
                                <rect>
                                    <x>0</x>
                                    <y>230</y>
                                    <width>120</width>
                                    <height>331</height>
                                </rect>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>120</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>120</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="verticalScrollBarPolicy">
                                <enum>Qt::ScrollBarAlwaysOff</enum>
                            </property>
                            <property name="horizontalScrollBarPolicy">
                                <enum>Qt::ScrollBarAlwaysOff</enum>
                            </property>
                            <property name="sizeAdjustPolicy">
                                <enum>QAbstractScrollArea::AdjustIgnored</enum>
                            </property>
                            <item>
                                <property name="text">
                                    <string>新建项目</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>新建项目</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>新建项目</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>新建项目</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>新建项目</string>
                                </property>
                            </item>
                        </widget>
                    </widget>
                </item>
                <item>
                    <widget class="QStackedWidget" name="stackedWidget">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <weight>50</weight>
                                <bold>false</bold>
                            </font>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1280</width>
                    <height>23</height>
                </rect>
            </property>
            <widget class="QMenu" name="menu_F">
                <property name="title">
                    <string>文件(F)</string>
                </property>
                <addaction name="separator"/>
                <addaction name="separator"/>
                <addaction name="action_3"/>
                <addaction name="action_4"/>
            </widget>
            <widget class="QMenu" name="menu_data">
                <property name="font">
                    <font>
                        <family>微软雅黑</family>
                    </font>
                </property>
                <property name="title">
                    <string>数据</string>
                </property>
                <widget class="QMenu" name="menu_output">
                    <property name="title">
                        <string>导出聊天记录(全部)</string>
                    </property>
                    <addaction name="action_output_CSV"/>
                </widget>
                <addaction name="menu_output"/>
            </widget>
            <widget class="QMenu" name="menu_2">
                <property name="title">
                    <string>帮助</string>
                </property>
                <addaction name="action_help_decrypt"/>
                <addaction name="action_help_chat"/>
                <addaction name="action_help_contact"/>
            </widget>
            <widget class="QMenu" name="menu_about">
                <property name="title">
                    <string>关于</string>
                </property>
                <addaction name="action_desc"/>
            </widget>
            <widget class="QMenu" name="menu_3">
                <property name="title">
                    <string>不显示或者显示异常请重启应用、没反应那就多等一会儿</string>
                </property>
            </widget>
            <addaction name="menu_F"/>
            <addaction name="menu_data"/>
            <addaction name="menu_2"/>
            <addaction name="menu_about"/>
            <addaction name="menu_3"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="action_3">
            <property name="text">
                <string>保存</string>
            </property>
        </action>
        <action name="action_4">
            <property name="text">
                <string>退出</string>
            </property>
        </action>
        <action name="action_help_decrypt">
            <property name="text">
                <string>解密教程</string>
            </property>
            <property name="font">
                <font>
                    <family>Microsoft YaHei UI</family>
                </font>
            </property>
        </action>
        <action name="action_desc">
            <property name="text">
                <string>说明</string>
            </property>
        </action>
        <action name="action_help_chat">
            <property name="text">
                <string>聊天相关</string>
            </property>
        </action>
        <action name="action_help_contact">
            <property name="text">
                <string>好友相关</string>
            </property>
        </action>
        <action name="action_output_CSV">
            <property name="text">
                <string>CSV</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
